### PR TITLE
RT2.6,RT2.8,RT2.9-modified scripts to use session.go from the internal folder

### PR DESCRIPTION
**Pull request by divyarema**
Modified Rt2.6,RT2.8,RT2.9 to use session.go from the internal folder

### CODE REVIEW

**Review by sachendras — DISMISSED** [review]

File: feature/experimental/isis/otg_tests/isis_interface_hello_padding_enable_test/isis_interface_hello_padding_enable_test.go [hunk 1]
```diff
@@ -41,102 +36,40 @@ func TestMain(m *testing.M) {
...
-	netInstance := d.GetOrCreateNetworkInstance(deviations.DefaultNetworkInstance(dut))
-	prot := netInstance.GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_ISIS, isisInstance)
+	d := ts.DUTConf
+	netInstance := d.GetOrCreateNetworkInstance(deviations.DefaultNetworkInstance(ts.DUT))
+	prot := netInstance.GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_ISIS, session.ISISName)
 	prot.Enabled = ygot.Bool(true)
+
 	isis := prot.GetOrCreateIsis()
-	globalISIS := isis.GetOrCreateGlobal()
+	globalIsis := isis.GetOrCreateGlobal()
```
Comment (greg-dennis) [inline feature/experimental/isis/otg_tests/isis_interface_hello_padding_enable_test/isis_interface_hello_padding_enable_test.go:65]: "globalIsis" -> "globalISIS"

File: feature/experimental/isis/otg_tests/isis_metric_style_wide_not_enabled_test/isis_metric_style_wide_not_enabled_test.go [hunk 2]
```diff
@@ -41,109 +35,49 @@ func TestMain(m *testing.M) {
...
-	netInstance := d.GetOrCreateNetworkInstance(deviations.DefaultNetworkInstance(dut))
-	prot := netInstance.GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_ISIS, isisInstance)
+	d := ts.DUTConf
+	netInstance := d.GetOrCreateNetworkInstance(deviations.DefaultNetworkInstance(ts.DUT))
+	prot := netInstance.GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_ISIS, session.ISISName)
 	prot.Enabled = ygot.Bool(true)
 
 	isis := prot.GetOrCreateIsis()
-	globalISIS := isis.GetOrCreateGlobal()
+	globalIsis := isis.GetOrCreateGlobal()
```
Comment (greg-dennis) [inline feature/experimental/isis/otg_tests/isis_metric_style_wide_not_enabled_test/isis_metric_style_wide_not_enabled_test.go:68]: "globalIsis" -> "globalISIS"

File: feature/experimental/isis/otg_tests/isis_metric_style_wide_enabled_test/isis_metric_style_wide_enabled_test.go [hunk 3]
```diff
@@ -41,99 +35,39 @@ func TestMain(m *testing.M) {
...
-	configPath := gnmi.OC().NetworkInstance(deviations.DefaultNetworkInstance(dut)).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_ISIS, isisInstance)
-	netInstance := d.GetOrCreateNetworkInstance(deviations.DefaultNetworkInstance(dut))
-	prot := netInstance.GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_ISIS, isisInstance)
+	d := ts.DUTConf
+	netInstance := d.GetOrCreateNetworkInstance(deviations.DefaultNetworkInstance(ts.DUT))
+	prot := netInstance.GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_ISIS, session.ISISName)
 	prot.Enabled = ygot.Bool(true)
+
 	isis := prot.GetOrCreateIsis()
 	globalIsis := isis.GetOrCreateGlobal()
```
Comment (greg-dennis) [inline feature/experimental/isis/otg_tests/isis_metric_style_wide_enabled_test/isis_metric_style_wide_enabled_test.go:68]: "globalIsis" -> "globalISIS"

**Review by greg-dennis — DISMISSED** [review]

File: feature/experimental/isis/otg_tests/isis_interface_hello_padding_enable_test/isis_interface_hello_padding_enable_test.go (hunk 1 above)
Comment (divyarema) [reply feature/experimental/isis/otg_tests/isis_interface_hello_padding_enable_test/isis_interface_hello_padding_enable_test.go]: comments addressed

File: feature/experimental/isis/otg_tests/isis_metric_style_wide_enabled_test/isis_metric_style_wide_enabled_test.go (hunk 3 above)
Comment (divyarema) [reply feature/experimental/isis/otg_tests/isis_metric_style_wide_enabled_test/isis_metric_style_wide_enabled_test.go]: comments addressed

File: feature/experimental/isis/otg_tests/isis_metric_style_wide_not_enabled_test/isis_metric_style_wide_not_enabled_test.go (hunk 2 above)
Comment (divyarema) [reply feature/experimental/isis/otg_tests/isis_metric_style_wide_not_enabled_test/isis_metric_style_wide_not_enabled_test.go]: comments addressed

**Review by arulkumarsekar — APPROVED** [review]

**Pull request merged**